sdk/communication/azure-communication-chat/README.md

```diff
@@ -0,0 +1,714 @@
+# Azure Communication Chat Service client library for Android
```

Comment:
Is it Chat **Service** or Chat **Services**?

Reply:
"Services" were used in other places and I believe we should be consistent

Reply:
In this case, singular makes sense